app/public/templates/scene.html

```diff
@@ -139,7 +139,7 @@
           const next_scene_idx = {{ cur_scene_idx }} + 1;
 
           if({{cur_scene_idx }} == 0){
-            const intro_transition_data = {% raw json.dumps(scenario_transitions[0]) %};
+            const intro_transition_data = {% raw json_encode(scenario_transitions[0]) %};
```

Comment:
In the above two cases, you removed the `% raw` but here (and below) it is kept. Any reason?

Reply:
Yeah, those 2 are in HTML context so we need it to be escaped so HTML will read it properly. In these later ones, we are in the "script" tag, so it's interpreted as JS, so need to have it as is (hence raw)